### ISSUE TITLE
I cant install library(dashDaq) in R

### ISSUE THREAD

**Issue by yuliaUU** [plotly/dash-daq#101]
When I run `install.packages(dashDaq)`

I get an error : `package dashDaq is not available for R version 3.6.3`

**Comment by rpkyle** [comment]
> When I run `install.packages(dashDaq)`
> 
> I get an error : `package dashDaq is not available for R version 3.6.3`

Hi @yuliaUU, `dashDaq` isn’t on CRAN yet, so you’ll want to install using `install_github` from the `devtools` package, e.g.

```
library(devtools)
install_github(“plotly/dash-daq”, upgrade=TRUE)
```

**Comment by yuliaUU** [comment]
Thanks a ton! I just realized that too! Thank you for your fast reply
